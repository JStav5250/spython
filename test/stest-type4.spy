int x = 3
type(x)

str x = "hello"
type(x)

float x = 3.0
type(x)

bool x = true
type(x)

x = false
type(x)

