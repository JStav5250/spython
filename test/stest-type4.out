int
str
float
bool
bool
